api.py

```
@@ -624,45 +633,72 @@
+                        # Generate status response
+                        response = create_task_status_response(celery_task, task, task_id, base_url)
+                        data = f"data: {json.dumps(response)}\n"  # Note the double newline for SSE format
+                        
+                        
+                        yield data.encode('utf-8')  # Ensure we're yielding bytes
+                        
+                        # Exit when the task is complete
+                        if celery_task_ready:
+                            break
+                            
+                    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)